doc/Doxyfile.in

```diff
@@ -31,10 +31,18 @@ MULTILINE_CPP_IS_BRIEF = NO
 INHERIT_DOCS           = YES
 SEPARATE_MEMBER_PAGES  = NO
 TAB_SIZE               = 4
-ALIASES                =
+ALIASES               += addhtml{1}="@htmlonly[block] <div class='newInnerHTML'>\1</div><div> @endhtmlonly"
+ALIASES               += add_toggle{1}="@addhtml{\1} @htmlinclude add_toggle.html \parblock \endparblock"
+ALIASES               += end_toggle="@htmlonly[block] </div> @endhtmlonly"
+ALIASES               += prev_tutorial{1}="**Prev  Tutorial:** \ref \1 \n"
+ALIASES               += next_tutorial{1}="**Next  Tutorial:** \ref \1 \n"
+ALIASES               += add_youtube_link{1}="@htmlonly[block] <div class='addYoutube'>\1</div> @endhtmlonly"
+ALIASES               += youtube{1}="@add_youtube_link{\1} @htmlinclude add_youtube.html \parblock \endparblock"
```

Comment:
Ok, so it's more important to keep the /doc directory clean rather than the Alias file itself?